TuviSRPLib/TuviSRPLib/ExtendedHashDigest.cs

```diff
@@ -40,17 +41,17 @@ public void BlockUpdate(byte[] input, int inOff, int length)
                 throw new ArgumentOutOfRangeException(nameof(inOff), "Parameter inOff can not be negative.");
             }
 
-            if (length < 0)
+            if (inLen < 0)
             {
-                throw new ArgumentOutOfRangeException(nameof(length), "Parameter length can not be negative.");
+                throw new ArgumentOutOfRangeException(nameof(inLen), "Parameter length can not be negative.");
```

Comment:
`Parameter length can not be negative.`
length should be replaced with inLen

Reply:
done